wp-web-push/wp-web-push-main.php

```diff
@@ -3,11 +3,18 @@
 require_once(plugin_dir_path(__FILE__) . 'web-push.php' );
 require_once(plugin_dir_path(__FILE__) . 'wp-web-push-db.php');
 
+load_plugin_textdomain('wpwebpush', false, dirname(plugin_basename(__FILE__)) . '/lang');
+
 class WebPush_Main {
   private static $instance;
+  public static $ALLOWED_TRIGGERS;
 
   public function __construct() {
     add_action('wp_head', array($this, 'add_manifest'));
+    self::$ALLOWED_TRIGGERS = array(
+      array('text' => __('New Post', 'wpwebpush'), 'key' => 'new-post', 'enable_by_default' => true, 'hook' => 'on_transition_post_status', 'action' => 'send_new_post')
```

Comment:
The hook should be `transition_post_status` and the action `on_transition_post_status`.

The other `add_action` using the same hook should be removed (it's a few lines after this change).